test/Libraries/IronPythonTests/README.md

```diff
@@ -0,0 +1,7 @@
+## IronPython Tests ##
+
+This project has been disabled from the main Dynamo solution because it Iron Python is no longer distributed with Dynamo and is no longer maintained.
```

Comment:
`... because IronPython is no longer ...`

Reply:
done